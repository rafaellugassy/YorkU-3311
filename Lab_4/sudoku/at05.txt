set_number(3, 2, 1)

-- Error: invalid number
set_number(5, 2, 2)
set_number(0, 2, 2)