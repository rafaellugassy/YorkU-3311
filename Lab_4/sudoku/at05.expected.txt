  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(3,2,1)
  Setting up a new grid
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(5,2,2)
  Error: Invalid value to put in cell
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(0,2,2)
  Error: Invalid value to put in cell
  0   0   0   0
  3   0   0   0
  0   0   0   0
  0   0   0   0
